includes/class-wc-cart.php

```diff
@@ -1172,6 +1172,7 @@ public function restore_cart_item( $cart_item_key ) {
 	 */
 	public function set_quantity( $cart_item_key, $quantity = 1, $refresh_totals = true ) {
 		if ( 0 === $quantity || $quantity < 0 ) {
+			wc_do_deprecated_action( 'woocommerce_before_cart_item_quantity_zero', array( $cart_item_key, $this ), '2.7.0', 'woocommerce_remove_cart_item' );
```

Comment:
Version should be `3.7.0`instead of `2.7.0`.

Reply:
Oh my bad. I've fixed that up.